src/rust/protocol/src/ping/wire/mod.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+mod request;
+mod response;
+
+pub use request::*;
+pub use response::*;
+
+use super::PingStorage;
+use crate::*;
+
+// use metrics::Stat;
```

Comment:
intentional? temporary?

Reply:
Temporary - need to figure out a reasonable way to approach stats when we have multiple servers sharing the same framework. Currently I'm sacrificing the pingserver specific stats and exposing segcache specific stats even for the pingserver. Not ideal, but I'll add a TODO if I can't come up with a clever hack.

Reply:
Adding a TODO for this. I think that fixing this is going to depend on having a better metrics registry where we don't need to define all the stats in one place.